app/views/conversations/_hat_inputs.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if user.wearable_hats.any? %>
```

Comment:
Note: unlikely to ever occur, but a moderator without wearable hats won't get the option to add a modnote

Reply:
Yup, I should use the `is_moderator` flag instead.